components/Layout/Layout.styled.js

```diff
@@ -7,14 +7,17 @@ export const NavBar = styled.nav`
   display: flex;
   justify-content: space-around;
   width: 100%;
+  border-top: 0.1px solid black;
+  border-top-left-radius: 20px; /* Add this line */
```

Comment:
I am not sure what the comments for for here?

If we don't need them i would remove them before merging into `main`

Reply:
Thank for your feedback I will remove it now